config/templates/cloud-config-worker

```diff
@@ -466,8 +465,6 @@ write_files:
            done;
            echo done.'
 
-      sudo rkt rm --uuid-file=/var/run/coreos/decrypt-tls-assets.uuid
```

Comment:
Could you share the context behind this? AFAIK, removing explicit `rkt rm`s just means to "defer" removals of rkt pods until `rkt gc` is run. Then, could there be a specific reason to do so?